### ISSUE TITLE
save mappoints

### ISSUE THREAD

**Issue by RIPGuy** [DioVei/ORB_SLAM3_with_save#2]
are they keypoints u mentioned here the same as mappoints?

**Comment by DioVei** [comment]
No, body, keypoints are image pixels, if you want 3d mappoints, just read pcd file directly.
